python-checks/src/main/java/org/sonar/python/checks/RedundantJumpCheck.java

```diff
@@ -49,22 +56,40 @@ public void initialize(Context context) {
   }
 
   private static void checkCfg(@Nullable ControlFlowGraph cfg, SubscriptionContext ctx) {
-    if (cfg == null) {
-      return;
-    }
-    for (CfgBlock cfgBlock : cfg.blocks()) {
-      if (cfgBlock.successors().size() == 1 && cfgBlock.successors().contains(cfgBlock.syntacticSuccessor())) {
+    Optional.ofNullable(cfg)
+      .map(ControlFlowGraph::blocks)
+      .stream()
+      .flatMap(Collection::stream)
+      .filter(cfgBlock -> cfgBlock.successors().size() == 1 && cfgBlock.successors().contains(cfgBlock.syntacticSuccessor()))
+      .forEach(cfgBlock -> {
         List<Tree> elements = cfgBlock.elements();
         Tree lastElement = elements.get(elements.size() - 1);
         if (!isException(lastElement)) {
-          PreciseIssue preciseIssue = ctx.addIssue(lastElement, message(lastElement));
+          var issue = ctx.addIssue(lastElement, message(lastElement));
+          addQuickFix(lastElement, issue);
+
           if (lastElement.is(Kind.CONTINUE_STMT)) {
             Tree loop = ((PythonCfgBranchingBlock) cfgBlock.successors().iterator().next()).branchingTree();
-            preciseIssue.secondary(loop.firstToken(), null);
+            issue.secondary(loop.firstToken(), null);
           }
         }
-      }
+      });
+  }
+
+  private static void addQuickFix(Tree lastElement, PreciseIssue issue) {
+    if (!(lastElement instanceof Statement)) {
+      return;
     }
+    Optional.of(issue)
+      .filter(IssueWithQuickFix.class::isInstance)
+      .map(IssueWithQuickFix.class::cast)
+      .ifPresent(i -> {
+          var quickFix = PythonQuickFix
```

Comment:
There is a code smell on that line due to inconsistent indentation.

Reply:
done